src/pattern-library/components/patterns/prototype/SelectPage.tsx

```diff
@@ -527,6 +492,35 @@ export default function SelectNextPage() {
           </Library.Example>
         </Library.Pattern>
       </Library.Section>
+
+      <Library.Section
+        title="MultiSelect"
+        intro={
+          <>
+            <p>
+              <code>MultiSelect</code> has the same public API as{' '}
+              <code>Select</code>, but allows multiple options to be selected at
```

Comment:
The public API is not _exactly_ the same, since `value` and `onChange` have different types. I would say something like, "MultiSelect is a variant of Select that allows multiple options to be selected at once. It has the same API except that `value` accepts, and `onChange` receives, arrays rather than a single item."